truckersmp-cli

```diff
@@ -1,512 +1,547 @@
+
+        if not os.path.isdir(args.protondir):
+            printdbg("Creating directory {}".format(args.protondir))
+            os.makedirs(args.protondir, exist_ok = True)
+
+        printdbg("""Command:
+  {}
+    +login {}
+    +force_install_dir {}
+    +app_update {} validate
+    +quit""".format(steamcmd, args.account, args.protondir, args.proton_appid))
+        subprocess.call([steamcmd,
+          "+login", args.account,
+          "+force_install_dir", args.protondir,
+          "+app_update", str(args.proton_appid), "validate",
+          "+quit"])
 
     # use steamcmd to update the chosen game
-    print "Updating $steamid"
-    print "Command:"
-    print   "$steamcmd" \
-                "+@sSteamCmdForcePlatformType windows" \
-                "+login $accountname" \
-                "+force_install_dir $gamedir" \
-                "+app_update $steamid validate" \
-                "+quit"
-
-    "$steamcmd" \
-        +@sSteamCmdForcePlatformType windows \
-        +login "$accountname" \
-        +force_install_dir "$gamedir" \
-        +app_update "$steamid" validate \
-        +quit
-}
-
-check_error() {
+    printdbg("Updating Game (AppId:{})".format(args.steamid))
+    printdbg("""Command:
+  {}
+    +@sSteamCmdForcePlatformType windows
+    +login {}
+    +force_install_dir {}
+    +app_update {} validate
+    +quit""".format(steamcmd, args.account, args.gamedir, args.steamid))
+    subprocess.call([steamcmd,
+      "+@sSteamCmdForcePlatformType", "windows",
+      "+login", args.account,
+      "+force_install_dir", args.gamedir,
+      "+app_update", args.steamid, "validate",
+      "+quit"])
+
+def check_error():
     # checks for updating and/or starting
-    if [ "$update" = false ] && [ "$startgame" = false ]; then
-        >&2 echo "What to do? Starting (-s) or updating (-u) the game?"
-        quit 1
-    else
-        # make sure only one game is chosen
-        if [ "$ats" = true ] && [ "$ets2" = true ]; then
-            >&2 echo "It's only possible to use one game at a time."
-            quit 1
-        elif ! [ "$ats" = true ] && ! [ "$ets2" = true ]; then
-            >&2 echo "Need at least one game. ATS (-a) or ETS2 (-e)"
-            quit 1
-        elif [ "$ats" = true ]; then
-            steamid="$appid_ats"
-            if [ -z "$prefixdir" ]; then
-                prefixdir="$default_prefixdir_ats"
-            fi
-            if [ -z "$gamedir" ]; then
-                gamedir="$default_gamedir_ats"
-            fi
-        elif [ "$ets2" = true ]; then
-            steamid="$appid_ets2"
-            if [ -z "$prefixdir" ]; then
-                prefixdir="$default_prefixdir_ets2"
-            fi
-            if [ -z "$gamedir" ]; then
-                gamedir="$default_gamedir_ets2"
-            fi
-        fi
-    fi
+    if not args.update and not args.start:
+        sys.exit("What to do? Starting (-s) or updating (-u) the game?")
```

Comment:
I feel like if the script receives zero arguments, we should show the usage string instead of this message.